resources/liftover.py

```diff
@@ -0,0 +1,192 @@
+    :return: StructExpression containing counts for mismatches and count for all variants on negative strand
+    :rtype: StructExpression
+    """
+
+    mismatch = ht.aggregate(hl.struct(total_mismatch=hl.agg.count_where(ht.reference_mismatch),
+                                negative_strand=hl.agg.count_where(ht.new_locus.is_negative_strand),
+                                negative_strand_mismatch=hl.agg.count_where(ht.new_locus.is_negative_strand & ht.reference_mismatch)
+                                )
+                            )
+    return mismatch
+
+
+def main(args):
+
+    hl.init()
+    
+    # get reference genomes
+    rg37 = hl.get_reference('GRCh37')
+    rg38 = hl.get_reference('GRCh38')
+
+    if args.gnomad:
+        gnomad = True
+        table_path = None
+        if not args.exomes and not args.genomes:
+            sys.exit('Error: Must specify --exomes or --genomes if lifting gnomAD ht')
+
+        if args.exomes:
+            data_type = 'exomes'
+        if args.genomes:
+            data_type = 'genomes'
+
+        logger.info('Working on gnomAD {} release ht'.format(data_type))
+        logger.info('Reading in release ht')
+        ht = get_gnomad_public_data(data_type, split=True, version=CURRENT_RELEASE)
+
+    else:
+        data_type = None
+        gnomad = None
+        table_path = args.table_path
+        ht = hl.read_table(table_path)
+   
+    if args.test: 
+        logger.info('Filtering to chr21 for testing')
+        if args.build == 38:
+            ht = ht.filter(ht.locus.contig == '21')
+        else:
+            ht = ht.filter(ht.locus/contig == 'chr21')
+
+    logger.info('Adding chain file and loading fasta sequence for destination build')
+    if args.build == 38:
+
+        rg37.add_liftover('gs://hail-common/references/grch37_to_grch38.over.chain.gz', rg38)
+        rg38.add_sequence(
+            'gs://hail-common/references/Homo_sapiens_assembly38.fasta.gz', 
+            'gs://hail-common/references/Homo_sapiens_assembly38.fasta.fai'
+            )
+
+        logger.info('Lifting ht to b{}'.format(args.build))
+        ht = lift_ht(ht, gnomad, data_type, table_path, rg38)
+        
+        logger.info('Checking SNPs for reference mismatches')
+        ht = annotate_snp_mismatch(ht, data_type, rg38)
+
+    else:
+        rg38.add_liftover('gs://hail-common/references/grch38_to_grch37.over.chain.gz', rg37)
+        rg37.add_sequence(
+            'gs://hail-common/references/human_g1k_v37.fasta.gz',
+            'gs://hail-common/references/human_g1k_v37.fasta.fai'
+            ) 
+
+        logger.info('Lifting ht to b{}'.format(args.build))
+        ht = lift_ht(ht, gnomad, data_type, table_path, rg37)
+
+        logger.info('Checking SNPs for reference mismatches')
+        ht = annotate_snp_mismatch(ht, data_type, rg37)
+
+    mismatch = check_mismatch(ht)
+    logger.info('{} reference mismatches in SNPs'.format(mismatch['total_mismatch']))
```

Comment:
Would be worth printing the full count and `'negative_strand'` count here as well